cfme/tests/cli/test_appliance_console.py

```diff
@@ -269,81 +276,88 @@ def is_storage_extended():
                             matched_patterns=['.*{} to false.*'.format(auth_type.option)],
-                            hostname=ipa_crud.hostname,
+                            hostname=configured_appliance.hostname,
                             username=app_creds['sshlogin'],
-                            password=app_creds['password'])
+                            password=app_creds['sshpass'])
 
     evm_tail.fix_before_start()
     command_set = ('ap', '', '12', auth_type.index, '4')
     ipa_crud.appliance_console.run_commands(command_set)
     evm_tail.validate_logs()
 
 
-@pytest.mark.uncollect('No IPA servers currently available')
-def test_appliance_console_external_auth_all(app_creds, ipa_crud):
-    """'ap' launches appliance_console, '' clears info screen, '12' change ext auth options,
+def test_appliance_console_external_auth_all(app_creds, ipa_crud, configured_appliance):
+    """'ap' launches appliance_console, '' clears info screen, '12/15' change ext auth options,
     'auth_type' auth type to change, '4' apply changes."""
 
     evm_tail = LogValidator('/var/www/miq/vmdb/log/evm.log',
-                            matched_patterns=['.*sso_enabled to true.*', '.*saml_enabled to true.*',
-                                '.*local_login_disabled to true.*'],
-                            hostname=ipa_crud.hostname,
+                            matched_patterns=['.*sso_enabled to true.*',
+                                              '.*saml_enabled to true.*',
+                                              '.*local_login_disabled to true.*'],
+                            hostname=ipa_crud.host1,
                             username=app_creds['sshlogin'],
                             password=app_creds['password'])
     evm_tail.fix_before_start()
-    command_set = ('ap', '', '12', '1', '2', '3', '4')
-    ipa_crud.appliance_console.run_commands(command_set)
+    ext_auth = '12'
+    command_set = ('ap', '', ext_auth, '1', '2', '3', '4')
+    configured_appliance.appliance_console.run_commands(command_set)
     evm_tail.validate_logs()
 
     evm_tail = LogValidator('/var/www/miq/vmdb/log/evm.log',
                             matched_patterns=['.*sso_enabled to false.*',
-                                '.*saml_enabled to false.*', '.*local_login_disabled to false.*'],
-                            hostname=ipa_crud.hostname,
+                                              '.*saml_enabled to false.*',
+                                              '.*local_login_disabled to false.*'],
+                            hostname=ipa_crud.host1,
                             username=app_creds['sshlogin'],
                             password=app_creds['password'])
 
     evm_tail.fix_before_start()
-    command_set = ('ap', '', '12', '1', '2', '3', '4')
-    ipa_crud.appliance_console.run_commands(command_set)
+    command_set = ('ap', '', ext_auth, '1', '2', '3', '4')
+    configured_appliance.appliance_console.run_commands(command_set)
```

Comment:
**QUESTION:** Shouldn't we assert on the return of the command being successful?

Reply:
Not possible in Appliance.ApplianceConsole.run_commands, unfortunately.